cmd/ttn-lw-cli/commands/contact_info.go

```diff
@@ -109,6 +109,7 @@ var contactInfoFlags = util.FieldFlags(&ttnpb.ContactInfo{})
 var (
 	errContactInfoExists           = errors.DefineAlreadyExists("contact_info_exists", "contact info already exists")
 	errMatchingContactInfoNotFound = errors.DefineAlreadyExists("contact_info_not_found", "matching contact info not found")
+	errMissingParam                = errors.DefineInvalidArgument("missing_params", "missing token or reference")
```

Comment:
Create `errNoReference` and `errNoValidationToken` instead

Reply:
Change error name accordingly